apps/client/src/views/admin/ListUser.vue

```diff
@@ -0,0 +1,47 @@
+<script setup>
+import { onMounted, ref } from 'vue'
+import { useAdminUserStore } from '@/stores/admin/user.js'
+
+const adminUserStore = useAdminUserStore()
+
+const allUsers = ref([])
+
+const title = 'Liste des utilisateurs'
+const headers = [
+  'Prénom',
+  'Nom',
+  'Courriel',
+  'Création',
+  'Modification',
+]
+const rows = ref([])
+
+const getAllUsers = async () => {
+  await adminUserStore.getAllUsers().then(response => {
+    allUsers.value = response
+  })
+}
+
+onMounted(async () => {
+  await getAllUsers()
+  allUsers.value.forEach(element => {
+    element.createdAt = formatDate(element.createdAt)
+    element.updatedAt = formatDate(element.updatedAt)
+    delete element.id
+    rows.value.push(element)
+  })
+})
```

Comment:
quand tu veux itérer et renvoyer le résultat tu peux faire un map, du genre

```js
const rows.value = allUsers.value?.map(element => ({
    firstName: element.firstName,
    lastName: element.lastName,
    email: element.email,
    createdAt: formatDate(element.createdAt),
    updatedAt: formatDate(element.updatedAt),
  }))
 ```

Reply:
ça te permet aussi de contrôler ce que tu mets dans tes `rows` (si jamais l'objet user change)